src/views/CoCurricularTranscript/guestTranscript.tsx

```diff
@@ -0,0 +1,116 @@
+import { MembershipHistory, MembershipHistorySession } from 'services/user';
+import { StudentProfileInfo } from 'services/user';
+import { Class } from 'services/peopleSearch';
+import { Participation } from 'services/membership';
+import { TranscriptItems } from 'services/transcript';
+
+const participation1: Participation = Participation.Member;
+const currentYear: number = new Date().getFullYear();
+const session1: MembershipHistorySession = {
+  MembershipID: -1,
+  SessionCode: currentYear - 1 + '09',
+  Participation: participation1,
+};
+const activity1: MembershipHistory = {
+  ActivityCode: 'ISO',
+  ActivityDescription: 'International Student Organization',
+  ActivityImagePath: '',
+  Sessions: [session1],
+  LatestDate: currentYear - 1 + '-12-15T00:00:00',
+};
+const participation2: Participation = Participation.Member;
+const session2: MembershipHistorySession = {
+  MembershipID: -1,
+  SessionCode: currentYear + '01',
+  Participation: participation2,
+};
+const activity2: MembershipHistory = {
+  ActivityCode: '360',
+  ActivityDescription: '360.gordon.edu',
+  ActivityImagePath: '',
+  Sessions: [session2],
+  LatestDate: currentYear + '-05-10T00:00:00',
+};
+export const transcriptItems1: TranscriptItems = {
+  honors: [],
+  experiences: [],
+  service: [],
+  activities: [activity1, activity2],
+};
+export const student1: StudentProfileInfo = {
+  fullName: 'Example Student',
+  Majors: ['Undecided'],
+  Minors: [],
+  Advisors: [],
+  ID: '12345678',
+  Title: '',
+  FirstName: 'Example',
+  MiddleName: '',
+  LastName: 'Student',
+  Suffix: '',
+  MaidenName: '',
+  NickName: '',
+  OnCampusBuilding: 'WIL',
+  OnCampusRoom: '000',
+  OnCampusPhone: '',
+  OnCampusPrivatePhone: '',
+  OnCampusFax: '',
+  Mail_Location: '000',
+  HomeStreet1: '255 Grapevine Rd',
+  HomeStreet2: '',
+  HomeCity: 'Wenham',
+  HomeState: 'MA',
+  HomePostalCode: '',
+  HomeCountry: 'US',
+  HomePhone: '',
+  HomeFax: '',
+  KeepPrivate: '',
+  Barcode: '12312312312312',
+  Email: 'first.last@gordon.edu',
+  Gender: 'F',
+  AD_Username: 'first.last',
```

Comment:
If you change the other names, these should also be example.student technically.

Reply:
Right! I forgot them